live/engine/account.py

```
@@ -259,3 +259,47 @@ def read_account(funder: str, collateral_usd: Optional[float],
+    if not (os.environ.get("POLY_PRIVATE_KEY") or os.environ.get("POLY_KEY")):
+        return None
+    try:
+        from py_clob_client_v2.clob_types import AssetType, BalanceAllowanceParams
+        from engine.venue import client
+        who = funder or os.environ.get("POLY_FUNDER")
+        sig_type = int(os.environ.get("POLY_SIG_TYPE", "3"))
+        r = client(who).get_balance_allowance(
+            BalanceAllowanceParams(asset_type=AssetType.COLLATERAL,
+                                   signature_type=sig_type))
+        return float(r.get("balance", 0) or 0) / 1e6
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)